### PR TITLE
Created Patient List , items checkable.

### PR DESCRIPTION
**Pull request by raresmanea**

### CODE REVIEW

File: webapp/controller/MainPage.controller.js [hunk 1]
```diff
@@ -11,6 +11,9 @@ sap.ui.define(['sap/ui/core/mvc/Controller', 'sap/ui/model/json/JSONModel', "com
 			var oModel = new JSONModel(this._data);
 			this.getView().setModel(oModel, 'systemDate');
 
+			let oPacientModel = models.createPacientModel();
+			this.getView().setModel(oPacientModel, "patientList");
+
 		},
 		onSelectedDoctor: function (oEvent) {
 			let currentDoctorPath = oEvent.getSource().getBindingContextPath("doctors"); debugger;
```
Comment (vlad9564) [inline webapp/controller/MainPage.controller.js]: remove the dubber in the code

File: webapp/view/MainPage.view.xml [hunk 2]
```diff
@@ -16,19 +17,50 @@
...
+									<Text class="headerList globalMargin" textAlign="Center" text="Patient List"/>
+								</HBox>
+								<HBox>
+									<List items="{path: 'patientList>/Patients'}" class="listSizing globalMargin">
+										<CustomListItem type="Navigation">
+											<CheckBox text="{patientList>id} {patientList>firstName} {patientList>lastName}" selected="false" valueState="Information"/>
+											<!-- <HBox>
+												<VBox class="sapUiSmallMarginBegin sapUiSmallMarginTopBottom">
+													<Label text="{patientList>id}"/>
+													<Label text="{patientList>name}"/>
```
Comment (vlad9564) [inline webapp/view/MainPage.view.xml]: Do you need the commented code?

**Review by vlad9564 — CHANGES_REQUESTED** [review]

File: webapp/css/style.css [hunk 3]
```diff
@@ -1 +1,17 @@
...
+}
+
+.globalMargin {
+    margin: 0rem 6rem;
+}
+
+.headerList {
+    width: 9rem !important;
+    font-size: 1.5rem;
+    padding: 2%;
```
Comment (razvanpan) [inline webapp/css/style.css]: Let's be consistent and use only one metrics for styling. In this case use or rem or %.

File: webapp/model/models.js [hunk 4]
```diff
@@ -13,7 +13,153 @@ sap.ui.define([
...
+						"lastName": "Predescu",
+						"age": "10",
+						"doctor": ""
+					}
+				]
+			};
+
+
+			let oModel = new JSONModel(oData);
+			return oModel;
```
Comment (razvanpan) [inline webapp/model/models.js]: Does not need to be a let here, should be a const. Put these two lines in one.

File: webapp/controller/MainPage.controller.js [hunk 5]
```diff
@@ -11,6 +11,9 @@ sap.ui.define(['sap/ui/core/mvc/Controller', 'sap/ui/model/json/JSONModel', "com
 			var oModel = new JSONModel(this._data);
```
Comment (razvanpan) [inline webapp/controller/MainPage.controller.js]: Make use of ES6 here (!var).

**Review by vlad9564 — APPROVED** [review]

**Pull request merged**